src/hermes/commands/deposit/invenio.py

```diff
@@ -252,14 +252,14 @@ class InvenioDepositPlugin(BaseDepositPlugin):
     invenio_resolver_class = InvenioResolver
     settings_class = InvenioDepositSettings
 
-    def __init__(self, click_ctx: click.Context, ctx: CodeMetaContext, client=None, resolver=None) -> None:
-        super().__init__(click_ctx, ctx)
+    def __init__(self, ctx: CodeMetaContext, client=None, resolver=None) -> None:
+        super().__init__(ctx)
 
         self.invenio_context_path = ContextPath.parse(f"deposit.{self.platform_name}")
         self.invenio_ctx = None
 
         if client is None:
-            auth_token = self.click_ctx.params.get("auth_token")
+            auth_token = self.ctx.params.get("auth_token")
```

Comment:
**chore:** We need to decide on how to pass command line arguments to the plugins. Currently, the only thing that plugins can interact with the "outside" world is the settings. Hence, I'd opt for putting everything that is needed by the plugin in to a setting. You can also configure those settings from command line by using e.g,, ``hermes harvest -O invenio.auth_tokten MY_AUTH_TOKEN -O invenio.initial True``